cartographer_ros/configuration_files/turtlebot.lua

```diff
@@ -12,27 +12,28 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 
-include "trajectory_builder.lua"
-include "sparse_pose_graph.lua"
+include "map_builder.lua"
 
 options = {
-  sparse_pose_graph = SPARSE_POSE_GRAPH,
-  trajectory_builder = TRAJECTORY_BUILDER,
+  map_builder = MAP_BUILDER,
   map_frame = "map",
-  odom_frame = "odom",
   tracking_frame = "base_link",
+  odom_frame = "odom",
   provide_odom_frame = false,
-  expect_odometry_data = false,
+  use_odometry_data = false,
   publish_occupancy_grid = false,
-  laser_min_range = 0.,
-  laser_max_range = 30.,
-  laser_missing_echo_ray_length = 5.,
-  lookup_transform_timeout = 0.01,
+  use_horizontal_laser = false,
```

Comment:
Shouldn't this be true and the multi echo false? At least this is how it was before.